### ISSUE TITLE
configuration for dataloaders

### ISSUE THREAD

**Issue by oubino** [oubino/locpix_points#5]
```
    # TODO: configuration for dataloaders
```

The dataloaders have args: batch_size, shuffle etc. 
This should be specified in the config file

https://github.com/oubino/heptapods/blob/a6a99e1e9012158711f38a8ea87f0046909c921f/recipes/train.py#L72
